src/ucp/proto/proto_am.inl

```diff
@@ -22,6 +22,15 @@
 #define UCP_STATUS_PENDING_SWITCH (UCS_ERR_LAST - 1)
 
 
+/**
+ * AM specific internal request flags
+ * These flags extend @ref ucp_send_am_flags enumeration
+ */
+enum ucp_request_am_internal_flags {
+    UCP_REQUEST_AM_FLAG_HEADER_SENT = UCS_BIT(15)
```

Comment:
we have to make sure it is not packed to the AM header

Reply:
reverted to separate `uint8_t internal_flags`